integration/query_test.go

```diff
@@ -235,6 +301,7 @@ func TestQueryMaxTimeMSErrors(t *testing.T) {
 				Name:    "BadValue",
 				Message: "maxTimeMS has non-integral value",
 			},
+			altMessage: "maxTimeMS has non-integral value",
```

Comment:
This `altMessage` is identical to the error message.

```suggestion
```

Reply:
Also for the same reason https://github.com/FerretDB/FerretDB/pull/2806/files#r1222350133